### PR TITLE
Chore/update wasabi binaries

### LINKED ISSUE #6938: Missing anonymity level in transaction detail
All my inputs and outputs should have attached calculated anonymity level.
screenshot_2022-11-16_at_18 20 33

[_link to original post_](https://satoshilabs.slack.com/archives/C048XFAKL10/p1668620508498339?thread_ts=1668618541.854499&cid=C048XFAKL10)

### LINKED ISSUE #6950: Repair absurdly high `anonymity level`
screenshot_2022-11-17_at_13 09 49

Screenshot 2022-11-17 at 13 14 24

[trezor-suite-log-1668686429534.txt](https://github.com/trezor/trezor-suite/files/10054534/trezor-suite-log-1668686429534.txt)

### PR DESCRIPTION
**Pull request by szymonlesisz**
## Description

update binaries from commit: https://github.com/trezor/coinjoin-backend/commit/e7ea05c517665bd38679aff7f3546bb33ce8f3e5

binary changes:
- added `/get-version` endpoint
- fixed memory leak in `/select-utxo-for-round` results with timeout
- fixed memory leak in `/analyze-transaction` fix https://github.com/trezor/trezor-suite/issues/6950 and fix https://github.com/trezor/trezor-suite/issues/6938

suite changes:
- use and log `/get-version` in CoinjoinProcess.status
- create and use `getPort` method of CoinjoinProcess
- create and use `getUrl` method of CoinjoinProcess

### CODE REVIEW

File: packages/suite-data/files/bin/coinjoin/update.sh [hunk 1]
```diff
@@ -10,8 +10,10 @@ BINARY_NAME=WalletWasabi.WabiSabiClientLibrary
 7z x -y CoinjoinClientLibrary-binary.zip
 
 # Re-naming direcotry from osx to mac to match the naming using in all the processes.
```
Comment (mroz22) [inline packages/suite-data/files/bin/coinjoin/update.sh]: direcotry :sob:

**Review by mroz22 — APPROVED** [review]
ok. 
I liked it more when getPort returned something and startOnPart accepted something. Feels more readable to me rather than having side effects. Probably just personal preference.

**Comment by szymonlesisz** [comment]
not sure what side effects you are talking about, but i needed to assing `CoinjoinProcess.port` **before** `CoinjoinProcess.startOnPort` is called to generate valid url, therefore assigning port again `CoinjoinProcess.startOnPort` doesnt make sesne anymore.

i was even thinking to rename it to `CoinjoinProcess.start` but didnt do it

**Comment by matejkriz** [comment]
Mac binaries are not signed, right? @vdovhanych Could you please sign them? 🙏

**Comment by szymonlesisz** [comment]
maybe we should sign them at build then?

**Comment by vdovhanych** [comment]
binaries signed for mac

**Comment by matejkriz** [comment]
> maybe we should sign them at build then?

related issue https://github.com/trezor/coinjoin-backend/issues/15

**Review by matejkriz — APPROVED** [review]

**Review by vdovhanych — APPROVED** [review]

**Pull request merged**